src/Core/src/Platform/iOS/MauiRefreshView.cs

```diff
@@ -178,7 +178,9 @@ public void UpdateIsEnabled(bool isRefreshViewEnabled)
 			UserInteractionEnabled = true;
 		}
 
-		bool CanUseRefreshControlProperty() =>
+#pragma warning disable CA1416 // TODO: 'UINavigationBar.PrefersLargeTitles' is only supported on: 'ios' 11.0 and later
+		bool CanUseRefreshControlProperty() => 
 			this.GetNavigationController()?.NavigationBar?.PrefersLargeTitles ?? true;
+#pragma warning restore CA1416
```

Comment:
Should this not just use a `=> IsAtLeast(11) && this.xxx.Prefers == true`?

Reply:
I can do that, I was not sure because that could cause behavioral change, it is returning true if its null, which is strange